crates/ruff_python_semantic/src/model.rs

```diff
@@ -2136,6 +2142,25 @@ bitflags! {
         /// while traversing the AST. (This only happens in stub files.)
         const DEFERRED_CLASS_BASE = 1 << 25;
 
+        /// The model is in an attribute docstring.
+        ///
+        /// An attribute docstring is a string literal immediately following an assignment or an
+        /// annotated assignment statement. The context in which this is valid are:
+        /// 1. At the top level of a module
+        /// 2. At the top level of a class definition i.e., a class attribute
+        ///
+        /// For example:
+        /// ```python
+        /// a = 1
+        /// """This is an attribute docstring for `a` variable"""
+        ///
+        ///
+        /// class Foo:
+        ///     b = 1
+        ///     """This is an attribute docstring for `Foo.b` class variable"""
+        /// ```
```

Comment:
```suggestion
        /// ```
        ///
        /// Unlike other kinds of docstrings, attribute docstrings are discarded at runtime.
        /// However, they are used by some documentation renderers and static-analysis tools.
```